src/plone/app/workflowmanager/browser/templates/controlpanel.pt

```diff
@@ -13,9 +13,14 @@
   <metal:css fill-slot="style_slot">
     <link rel="stylesheet" type="text/css" href="++resource++wfmbootstrap.css" media="screen" />
     <link rel="stylesheet" type="text/css" href="++resource++workflowmanager.css" media="screen" />
+    <link rel="stylesheet" type="text/css" href="++resource++workflow-graph.css" media="screen" />
   </metal:css>
   <metal:javascript fill-slot="javascript_head_slot">
     <script type="text/javascript" src="++resource++wfmbootstrap-popover.js"></script>
+    <script type="text/javascript" src="++resource++springy.js"></script>
```

Comment:
using `text/javascript` is redundant in HTML5 and must be omitted.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script